index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
Great work implementing most of the requested changes.
- [ ] As stated by the previous reviewer, Kindly make your design responsive, you will need to do so in order to have a professional-looking portfolio. 
Your design 
![image](https://user-images.githubusercontent.com/81932478/165802874-0916ee06-2f62-458c-b8bc-cdc029833b72.png)

Figma Design
![image](https://user-images.githubusercontent.com/81932478/165803169-88326a71-9331-4bb2-9066-aa77c557705f.png)

- The issue is that as you develop your site you are not using the inspect tool to see how your site looks like in mobile view. [Check out this video](https://www.youtube.com/watch?v=x8ofsJiELQ0)
- Once you have opened your site in mobile view use CSS to style your portfolio to match the Figma template you chose.
- Width properties and height properties should be used with caution they make your site to be unresponsive. For example, if you give the `header` section width of 300px, this means when viewing your site on mobile as the screen gets bigger than 300px the header will become static and white space will remain to cause the issue that you're having right now. I would advise removing all width and height properties or making all of them 100% as the previous reviewer advised. [See this article](https://www.w3schools.com/html/html_responsive.asp)